diff -up ./chirp/bitwise_grammar.py.orig ./chirp/bitwise_grammar.py
--- ./chirp/bitwise_grammar.py.orig	2011-04-23 19:43:34.000000000 -0400
+++ ./chirp/bitwise_grammar.py	2011-11-20 18:46:00.750416304 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/bitwise.py.orig ./chirp/bitwise.py
--- ./chirp/bitwise.py.orig	2011-08-17 07:00:16.000000000 -0400
+++ ./chirp/bitwise.py	2011-11-20 18:45:23.559707419 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/chirp_common.py.orig ./chirp/chirp_common.py
--- ./chirp/chirp_common.py.orig	2011-08-19 07:00:12.000000000 -0400
+++ ./chirp/chirp_common.py	2011-11-20 18:46:28.110465261 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/csv.py.orig ./chirp/csv.py
--- ./chirp/csv.py.orig	2011-08-26 07:00:10.000000000 -0400
+++ ./chirp/csv.py	2011-11-20 18:47:02.450271560 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/errors.py.orig ./chirp/errors.py
--- ./chirp/errors.py.orig	2011-08-29 07:00:12.000000000 -0400
+++ ./chirp/errors.py	2011-11-20 18:48:36.541000615 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ft2800.py.orig ./chirp/ft2800.py
--- ./chirp/ft2800.py.orig	2011-09-01 07:00:10.000000000 -0400
+++ ./chirp/ft2800.py	2011-11-20 18:54:35.839507259 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2011 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ft50_ll.py.orig ./chirp/ft50_ll.py
--- ./chirp/ft50_ll.py.orig	2011-04-23 19:43:34.000000000 -0400
+++ ./chirp/ft50_ll.py	2011-11-20 18:53:28.672843074 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ft50.py.orig ./chirp/ft50.py
--- ./chirp/ft50.py.orig	2011-04-23 19:43:34.000000000 -0400
+++ ./chirp/ft50.py	2011-11-20 18:52:59.224867119 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ft60.py.orig ./chirp/ft60.py
--- ./chirp/ft60.py.orig	2011-08-27 07:00:13.000000000 -0400
+++ ./chirp/ft60.py	2011-11-20 18:54:04.144609512 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2011 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ft7800.py.orig ./chirp/ft7800.py
--- ./chirp/ft7800.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/ft7800.py	2011-11-20 19:26:02.293946975 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ic2100.py.orig ./chirp/ic2100.py
--- ./chirp/ic2100.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/ic2100.py	2011-11-20 18:57:41.282057488 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ic2200.py.orig ./chirp/ic2200.py
--- ./chirp/ic2200.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/ic2200.py	2011-11-20 18:58:18.511762496 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ic2720.py.orig ./chirp/ic2720.py
--- ./chirp/ic2720.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/ic2720.py	2011-11-20 18:58:42.025944555 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2011 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ic2820.py.orig ./chirp/ic2820.py
--- ./chirp/ic2820.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/ic2820.py	2011-11-20 18:59:11.277927008 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ic9x_icf_ll.py.orig ./chirp/ic9x_icf_ll.py
--- ./chirp/ic9x_icf_ll.py.orig	2011-08-17 07:00:16.000000000 -0400
+++ ./chirp/ic9x_icf_ll.py	2011-11-20 18:56:41.827125473 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ic9x_icf.py.orig ./chirp/ic9x_icf.py
--- ./chirp/ic9x_icf.py.orig	2011-08-20 07:00:10.000000000 -0400
+++ ./chirp/ic9x_icf.py	2011-11-20 18:55:47.800004571 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ic9x_ll.py.orig ./chirp/ic9x_ll.py
--- ./chirp/ic9x_ll.py.orig	2011-05-27 07:00:12.000000000 -0400
+++ ./chirp/ic9x_ll.py	2011-11-20 18:57:13.477024627 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/ic9x.py.orig ./chirp/ic9x.py
--- ./chirp/ic9x.py.orig	2011-08-18 07:00:11.000000000 -0400
+++ ./chirp/ic9x.py	2011-11-20 18:55:19.099002772 -0500
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/icf.py.orig ./chirp/icf.py
--- ./chirp/icf.py.orig	2011-04-23 19:43:34.000000000 -0400
+++ ./chirp/icf.py	2011-11-20 18:59:34.132131991 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/icw32.py.orig ./chirp/icw32.py
--- ./chirp/icw32.py.orig	2011-08-27 07:00:13.000000000 -0400
+++ ./chirp/icw32.py	2011-11-20 19:01:01.796090357 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2011 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/icx8x_ll.py.orig ./chirp/icx8x_ll.py
--- ./chirp/icx8x_ll.py.orig	2011-06-09 07:00:12.000000000 -0400
+++ ./chirp/icx8x_ll.py	2011-11-20 19:02:02.924970034 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/icx8x.py.orig ./chirp/icx8x.py
--- ./chirp/icx8x.py.orig	2011-06-09 07:00:12.000000000 -0400
+++ ./chirp/icx8x.py	2011-11-20 19:01:29.096143449 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/id800.py.orig ./chirp/id800.py
--- ./chirp/id800.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/id800.py	2011-11-20 19:02:33.861896874 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/id880.py.orig ./chirp/id880.py
--- ./chirp/id880.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/id880.py	2011-11-20 19:03:02.587900366 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/idrp.py.orig ./chirp/idrp.py
--- ./chirp/idrp.py.orig	2011-08-19 07:00:12.000000000 -0400
+++ ./chirp/idrp.py	2011-11-20 19:03:53.110147636 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/import_logic.py.orig ./chirp/import_logic.py
--- ./chirp/import_logic.py.orig	2011-08-17 07:00:16.000000000 -0400
+++ ./chirp/import_logic.py	2011-11-20 19:04:14.711398200 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2011 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/__init__.py.orig ./chirp/__init__.py
--- ./chirp/__init__.py.orig	2011-09-13 18:35:50.000000000 -0400
+++ ./chirp/__init__.py	2011-11-20 19:22:09.001072389 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/kenwood_live.py.orig ./chirp/kenwood_live.py
--- ./chirp/kenwood_live.py.orig	2011-08-23 07:00:12.000000000 -0400
+++ ./chirp/kenwood_live.py	2011-11-20 19:05:10.143474923 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/memmap.py.orig ./chirp/memmap.py
--- ./chirp/memmap.py.orig	2011-04-23 19:43:34.000000000 -0400
+++ ./chirp/memmap.py	2011-11-20 19:06:17.105151403 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/platform.py.orig ./chirp/platform.py
--- ./chirp/platform.py.orig	2011-07-26 07:00:10.000000000 -0400
+++ ./chirp/platform.py	2011-11-20 19:06:56.395787945 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/thd72.py.orig ./chirp/thd72.py
--- ./chirp/thd72.py.orig	2011-06-09 07:00:12.000000000 -0400
+++ ./chirp/thd72.py	2011-11-20 19:08:26.203671154 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # ex: set tabstop=4 expandtab shiftwidth=4 softtabstop=4:
 #
 # Copyright 2010 Vernon Mauery <vernon@mauery.org>
diff -up ./chirp/tmv71_ll.py.orig ./chirp/tmv71_ll.py
--- ./chirp/tmv71_ll.py.orig	2011-04-23 19:43:34.000000000 -0400
+++ ./chirp/tmv71_ll.py	2011-11-20 19:09:07.989220848 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/tmv71.py.orig ./chirp/tmv71.py
--- ./chirp/tmv71.py.orig	2011-04-23 19:43:34.000000000 -0400
+++ ./chirp/tmv71.py	2011-11-20 19:08:46.991949639 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/bankedit.py.orig ./chirpui/bankedit.py
--- ./chirpui/bankedit.py.orig	2011-04-23 19:43:35.000000000 -0400
+++ ./chirpui/bankedit.py	2011-11-20 19:13:42.099704892 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/cloneprog.py.orig ./chirpui/cloneprog.py
--- ./chirpui/cloneprog.py.orig	2011-04-23 19:43:35.000000000 -0400
+++ ./chirpui/cloneprog.py	2011-11-20 19:14:24.743224178 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/clone.py.orig ./chirpui/clone.py
--- ./chirpui/clone.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirpui/clone.py	2011-11-20 19:14:03.328967758 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/common.py.orig ./chirpui/common.py
--- ./chirpui/common.py.orig	2011-08-26 07:00:10.000000000 -0400
+++ ./chirpui/common.py	2011-11-20 19:14:47.623429676 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/config.py.orig ./chirpui/config.py
--- ./chirpui/config.py.orig	2011-05-20 07:00:12.000000000 -0400
+++ ./chirpui/config.py	2011-11-20 19:15:19.835311096 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2011 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/dstaredit.py.orig ./chirpui/dstaredit.py
--- ./chirpui/dstaredit.py.orig	2011-04-23 19:43:35.000000000 -0400
+++ ./chirpui/dstaredit.py	2011-11-20 19:15:37.635692945 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/editorset.py.orig ./chirpui/editorset.py
--- ./chirpui/editorset.py.orig	2011-08-26 07:00:10.000000000 -0400
+++ ./chirpui/editorset.py	2011-11-20 19:15:57.499003140 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/importdialog.py.orig ./chirpui/importdialog.py
--- ./chirpui/importdialog.py.orig	2011-08-17 07:00:16.000000000 -0400
+++ ./chirpui/importdialog.py	2011-11-20 19:16:26.467997084 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/__init__.py.orig ./chirpui/__init__.py
--- ./chirpui/__init__.py.orig	2011-04-23 19:43:35.000000000 -0400
+++ ./chirpui/__init__.py	2011-11-20 19:16:52.749084344 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/inputdialog.py.orig ./chirpui/inputdialog.py
--- ./chirpui/inputdialog.py.orig	2011-04-23 19:43:35.000000000 -0400
+++ ./chirpui/inputdialog.py	2011-11-20 19:17:16.914244083 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/mainapp.py.orig ./chirpui/mainapp.py
--- ./chirpui/mainapp.py.orig	2011-09-01 07:00:10.000000000 -0400
+++ ./chirpui/mainapp.py	2011-11-20 19:17:44.679277260 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/memedit.py.orig ./chirpui/memedit.py
--- ./chirpui/memedit.py.orig	2011-08-26 07:00:10.000000000 -0400
+++ ./chirpui/memedit.py	2011-11-20 19:18:07.250491284 -0500
@@ -1,4 +1,3 @@
-#/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/reporting.py.orig ./chirpui/reporting.py
--- ./chirpui/reporting.py.orig	2011-04-30 07:00:14.000000000 -0400
+++ ./chirpui/reporting.py	2011-11-20 19:18:35.586504550 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2011 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirpui/shiftdialog.py.orig ./chirpui/shiftdialog.py
--- ./chirpui/shiftdialog.py.orig	2011-08-18 07:00:11.000000000 -0400
+++ ./chirpui/shiftdialog.py	2011-11-20 19:18:53.815869746 -0500
@@ -1,4 +1,3 @@
-#/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/util.py.orig ./chirp/util.py
--- ./chirp/util.py.orig	2011-04-23 19:43:34.000000000 -0400
+++ ./chirp/util.py	2011-11-20 19:09:46.657878649 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/vx3.py.orig ./chirp/vx3.py
--- ./chirp/vx3.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/vx3.py	2011-11-20 19:10:06.163201588 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2011 Rick Farina <sidhayn@gmail.com>
 #     based on modification of Dan Smith's original work
diff -up ./chirp/vx5.py.orig ./chirp/vx5.py
--- ./chirp/vx5.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/vx5.py	2011-11-20 19:10:23.401603200 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2011 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/vx6.py.orig ./chirp/vx6.py
--- ./chirp/vx6.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/vx6.py	2011-11-20 19:10:47.206776838 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/vx7.py.orig ./chirp/vx7.py
--- ./chirp/vx7.py.orig	2011-08-10 07:00:09.000000000 -0400
+++ ./chirp/vx7.py	2011-11-20 19:11:10.487968643 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/vx8.py.orig ./chirp/vx8.py
--- ./chirp/vx8.py.orig	2011-08-18 07:00:11.000000000 -0400
+++ ./chirp/vx8.py	2011-11-20 19:11:35.252108939 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/wouxun.py.orig ./chirp/wouxun.py
--- ./chirp/wouxun.py.orig	2011-09-09 07:00:13.000000000 -0400
+++ ./chirp/wouxun.py	2011-11-20 19:11:52.006527281 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2011 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/xml_ll.py.orig ./chirp/xml_ll.py
--- ./chirp/xml_ll.py.orig	2011-06-29 07:00:09.000000000 -0400
+++ ./chirp/xml_ll.py	2011-11-20 19:12:45.624665751 -0500
@@ -1,5 +1,3 @@
-#!/usr/bin/python
-#
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
 # This program is free software: you can redistribute it and/or modify
diff -up ./chirp/xml.py.orig ./chirp/xml.py
--- ./chirp/xml.py.orig	2011-08-16 07:00:09.000000000 -0400
+++ ./chirp/xml.py	2011-11-20 19:12:14.080760920 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2008 Dan Smith <dsmith@danplanet.com>
 #
diff -up ./chirp/yaesu_clone.py.orig ./chirp/yaesu_clone.py
--- ./chirp/yaesu_clone.py.orig	2011-04-23 19:43:35.000000000 -0400
+++ ./chirp/yaesu_clone.py	2011-11-20 19:13:03.457046610 -0500
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 #
 # Copyright 2010 Dan Smith <dsmith@danplanet.com>
 #
